CHANGELOG.md

```diff
@@ -26,6 +26,7 @@ Features:
 - [#1340](https://github.com/rails-api/active_model_serializers/pull/1340) Add support for resource-level meta. (@beauby)
 
 Fixes:
+- [#1622](https://github.com/rails-api/active_model_serializers/pull/1622) Fix fragment caching inherited serializers to use distinct per-serializer caches. (@lserman)
```

Comment:
@groyoh No, not being too picky. The problem with not mentioning inheritance is that without it there fix is unclear.  btw, I've also struggled with what the description should be https://github.com/rails-api/active_model_serializers/pull/1622/files/038f3fc91d103aee0a14dc725b4856bf86a0a0ad#r57530684